src/content/docs/en/reference/configuration-reference.mdx

```diff
@@ -492,13 +492,17 @@ Specifies the directory in the build output where Astro-generated assets (bundle
 
 <p>
 
-**Type:** `string`<br />
+**Type:** `string | Record.<string, string>`<br />
 **Default:** `undefined`<br />
 <Since v="2.2.0" />
```

Comment:
Note to ourselves: this *property* was introduced in v2.2.0 but this *option* will be introduced much later. So we'll have to decide how to let people know, so that someone doesn't try to use this in an earlier version of Astro.

Reply:
Note also to ourselves: I think I have a solution I'm happy with. We can discuss to make sure!